demos/coreMQTT/mqtt_demo_mutual_auth.c

```diff
@@ -338,6 +364,11 @@ static uint16_t usSubscribePacketIdentifier;
  */
 static uint16_t usUnsubscribePacketIdentifier;
 
+/**
+ * @brief MQTT packet type expected to be received from the MQTT broker.
```

Comment:
Can we add a comment about how because the demo is qos1 and single threaded we will never received multiple packets per wait on MQTT_ProcessLoop?

Reply:
Yes. That makes sense. Thanks